server/test_test.go

```diff
@@ -133,6 +135,13 @@ func require_Len(t *testing.T, a, b int) {
 	}
 }
 
+func require_LessThan[T constraints.Ordered](t *testing.T, a, b T) {
```

Comment:
In a future release when Go 1.22 is out we should change this to be `cmp.Ordered` which would be included since Go 1.21? https://pkg.go.dev/cmp@master#Ordered